modules/core/02-client/keeper/grpc_query_test.go

```diff
@@ -400,6 +398,94 @@ func (suite *KeeperTestSuite) TestQueryConsensusStates() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestQueryConsensusStateHeights() {
+	var (
+		req                      *types.QueryConsensusStateHeightsRequest
+		expConsensusStateHeights = []types.Height{}
```

Comment:
This is very much so a mega nit:

Does this need to be assigned? Could it be declared just so there's consistency in the var block. E.g.

```
var (
	req                      *types.QueryConsensusStateHeightsRequest
	expConsensusStateHeights []types.Height
)
```

Reply:
yes, I've removed the assignment